proc_CPMG.py

```diff
@@ -30,9 +30,13 @@
     s = s['ph1',1].C
     s.mean('nScans')
     s.reorder('t2',first=True)
+    # I would call tE tau_e?
     echo_center = abs(s)['tE',0].argmax('t2').data.item()
     s.setaxis('t2', lambda x: x-echo_center)
-    s.rename('tE','nEchoes').setaxis('nEchoes',r_[1:nEchoes+1])
+    t_echo = # you need to set this to the time of an echo
+    #   (how long from one echo to the next)
+    #   this should be calculated from the ppg
+    s.setaxis('tE',t_echo*(1+r_[0:nEchoes]))
```

Comment:
this comment here specifically identifies where the problem with the units for ``tE`` is